### PR TITLE
Update a11y focal, add links to accounts

### PR DESCRIPTION
**Pull request by mbroz2**
@steven1046 is the new a11y focal and will need to provide a11y approvals

Add links to each approvers github account.

### CODE REVIEW

**Review by tevans78 — APPROVED** [review]

**Review by tevans78 — CHANGES_REQUESTED** [review]
People links are not right, should be https://github.com/steven1046

File: .github/ISSUE_TEMPLATE/feature.md [hunk 1]
```diff
@@ -50,17 +50,17 @@ Before Development Starts or 8 weeks before Onboarding
...
+- [ ] **Globalization** ([Sam Wong](https://github.com/orgs/OpenLiberty/people/samwatibm) - Liberty / Simy Cheeran - tWAS). Translation is complete or N/A. TVT - complete or N/A. Approver adds label focalApproved:globalization to the Epic in Github.
 
 "Design Approved" features
-- [ ] **Accessibility** - (G Scott Johnston). Accessibility testing is complete or N/A. Approver adds label focalApproved:accessibility to the Epic in Github.
-- [ ] **ID** - (Kareen Deen). Documentation work is complete or N/A . Approver adds label focalApproved:id to the Epic in Github.
-- [ ] **Performance** - (Jared Anderson). Performance testing is complete with no high severity defects or N/A . Approver adds label focalApproved:performance to the Epic in Github.
-- [ ] **Serviceability** - (Don Bourne). Serviceability has been addressed.
+- [ ] **Accessibility** - ([Steven Zvonek](https://github.com/orgs/OpenLiberty/people/steven1046)). Accessibility testing is complete or N/A. Approver adds label focalApproved:accessibility to the Epic in Github.
```
[the remark this answers is not in the document]
Comment (mbroz2) [reply .github/ISSUE_TEMPLATE/feature.md]: seems like it might depend on permissions; changed links to point directly at account

**Comment by gscottj** [comment]
Thanks!

**Review by tevans78 — APPROVED** [review]
Looks good

**Comment by mbroz2** [comment]
@tevans78 since our templates don't have copyright headers, I'm guessing the right thing to do here is to just force merge?

**Pull request merged**